pkg/apis/serving/v1beta1/component.go

```diff
@@ -0,0 +1,15 @@
+package v1beta1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Components interface is implemented by predictor, transformer and explainer
+// +kubebuilder:object:generate=false
+type Component interface {
+	GetContainer(metadata metav1.ObjectMeta, containerConcurrency *int64, config *InferenceServicesConfig) *v1.Container
```

Comment:
Now that containerconcurrency is in the component itself, do we need to pass this through?

Reply:
it is on the `ComponentExtensionSpec` not the model server spec, ModelServerSpec maps to the container.